pages/common/waifu2x-ncnn-vulkan.md

```diff
@@ -0,0 +1,17 @@
+# waifu2x-ncnn-vulkan
+
+> Image upscaler for manga/anime-style images using NCNN neural network framework.
+> Runs fast on Intel / AMD / Nvidia / Apple-Silicon with Vulkan API.
```

Comment:
Is this line necessary?

Reply:
My intention was to emphasize the need of having Vulkan compatible GPU in order to run this. For that I coped the second line from project's description.